src/context/products.tsx

```diff
@@ -78,11 +78,11 @@ const ProductsContext = createContext<{
   setListViewType: (viewType: string) => void;
   resolveCartId?: () => Promise<string | undefined>;
   refreshCart?: () => void;
-  addToCart?: (
+  addToCart: (
```

Comment:
The addToCart api needs to be fixed in separate PR
It doesn't return error in case of network issues

Reply:
All the addToCart changes are related to making `npm run dev` to be able to develop the functionality.